modules/system/classes/UpdateManager.php

```diff
@@ -951,11 +951,10 @@ protected function createSignature($data, $secret)
         return base64_encode(hash_hmac('sha512', http_build_query($data, '', '&'), base64_decode($secret), true));
     }
 
-    //
-    // Internals
-    //
-
-    protected function getMigrationTableName()
+    /**
+     * @return string
```

Comment:
It was an intentional change since it seemed odd to leave in a comment specifying the following as "Internals" after I opened it up to public access. Maybe it should still be marked that way, but there wasn't anything automated screwing with anything.

Reply:
Ah, I see. Didn't even catch the access change, clearing shouldn't be reviewing stuff late at night. 

I'm on vacation until the 28th, ping me again later if I haven't taken another look by then.

Reply:
Yep, no worries. I'll keep an eye on this if it's still hanging around after your vacation. Thanks for letting me know.